### PR TITLE
Task nxp 26970 3d polymer 3

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: addons/nuxeo-platform-3d/elements/nuxeo-3d-viewer.js [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+Contributors:
+    Tiago Cardoso <tcardoso@nuxeo.com>
+    Miguel Nixo <mnixo@nuxeo.com>
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
+import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-resizable-behavior.js';
+
+import * as THREE from 'three';
```
Comment (Gabez0r) [inline addons/nuxeo-platform-3d/elements/nuxeo-3d-viewer.js]: I guess we don't need this, given that we're doing:
```
    plugins: [
      new ProvidePlugin({
        THREE: 'three'
      })
    ]
```
?

File: addons/nuxeo-platform-3d/elements/nuxeo-3d-render-views.js [hunk 2]
```diff
@@ -67,10 +68,10 @@ Example - Load a ThreeD document:
       <div class="render-views">
         <template is="dom-repeat" items="[[document.properties.threed:renderViews]]" as="renderView">
           <div class="item">
-            <img src="[[renderView.thumbnail.data]]" on-tap="_setCoords"/>
+            <img src="[[renderView.thumbnail.data]]" on-tap="_setCoords">
```
Comment (nmpcunha) [inline addons/nuxeo-platform-3d/elements/nuxeo-3d-render-views.js]: Should we use tags this way?

**Comment by Gabez0r** [comment]
👍

**Comment by nmpcunha** [comment]
👍

File: addons/nuxeo-platform-3d/elements/nuxeo-3d-render-views.js (hunk 2 above)
Comment (nelsonsilva) [reply addons/nuxeo-platform-3d/elements/nuxeo-3d-render-views.js]: The  tag is void element => can't have children so it shouldn't be closed.
Anyway this is just another artifact of the automatic polymer 3 migration 🤷‍♂️

**Pull request merged**